spanner/spanner_samples.rb

```diff
@@ -222,6 +222,53 @@ def create_database_with_encryption_key project_id:, instance_id:, database_id:,
   # [END spanner_create_database_with_encryption_key]
 end
 
+def create_database_with_multiple_kms_keys project_id:, instance_id:, database_id:, kms_key_names:
+  # [START spanner_create_database_with_MR_CMEK]
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+  # kms_key_names = "List of Database encryption KMS keys"
+
+  require "google/cloud/spanner"
+  require "google/cloud/spanner/admin/database"
+
+  database_admin_client = Google::Cloud::Spanner::Admin::Database.database_admin
+
+  instance_path = database_admin_client.instance_path project: project_id, instance: instance_id
+
+  db_path = database_admin_client.database_path project: project_id,
+                                                instance: instance_id,
+                                                database: database_id
+
+  job = database_admin_client.create_database parent: instance_path,
+                                              create_statement: "CREATE DATABASE `#{database_id}`",
+                                              extra_statements: [
+                                                "CREATE TABLE Singers (
+                                     SingerId     INT64 NOT NULL,
```

Comment:
Please keep the indentation consistent. For cases like this where you are embedding a multi-line string, consider defining the strings using heredocs. For example:

```
job = database_admin_client.create_database(
  parent: instance_path,
  create_statement: "CREATE DATABASE `#{database_id}`",
  extra_statements: [
    <<~STATEMENT,
      CREATE TABLE Singers (
        SingerId    INT64 NOT NULL,
        FirstName   STRING(1024),
        LastName    STRING(1024),
        SingerInfo  BYTES(MAX)
      ) PRIMARY KEY (SingerId)
    STATEMENT
    <<~STATEMENT
      CREATE TABLE Albums (
        SingerId    INT64 NOT NULL,
        AlbumId     INT64 NOT NULL,
        AlbumTitle  STRING(MAX)
      ) PRIMARY KEY (SingerId, AlbumId),
      INTERLEAVE IN PARENT Singers
      ON DELETE CASCADE
    STATEMENT
  ],
  encryption_config: { kms_key_names: kms_key_names }
)
```

Alternately, you can set the extra statement strings in temporary local variables like you do in the other sample below. Whatever is necessary to help readability for samples.